tensorflow/python/ops/math_ops.py

```diff
@@ -3305,6 +3305,15 @@ def log_sigmoid(x, name=None):
   Specifically, `y = log(1 / (1 + exp(-x)))`.  For numerical stability,
   we use `y = -tf.nn.softplus(-x)`.
 
+  Usage Example:
+   >>>import tensorflow as tf
```

Comment:
You can remove this import statement as it already exists in the file.

Reply:
Sorry about that, the new commit should solve these issues